app/cloudfoundry_config.py

```diff
@@ -14,6 +14,10 @@ def __init__(self):
             'region': ''
         }
 
+    @property
+    def database_url(self):
+        return getenv('DATABASE_URL', '').replace('postgres://', 'postgresql://')
```

Comment:
Does this need to be parsed similarly to Redis?

Reply:
No, because the RDS service sets `DATABASE_URL` when bound to an app. The redis service does not set anything equivalent, which is why you need to dig into `VCAP_SERVICES`

Reply:
[source](https://cloud.gov/docs/services/relational-database/#:~:text=In%20short%2C%20cf%20bind%2Dservice%20will%20provide%20a%20DATABASE_URL%20environment%20variable%20for%20your%20app)

Reply:
Cool, thanks for the link. That env var isn't in `cf env`, which is a little confusing, but I can confirm that it exists on a running version of the app.